component/azstorage/block_blob_test.go

```diff
@@ -3375,6 +3418,85 @@ func (s *blockBlobTestSuite) TestInvalidMD5OnReadNoVaildate() {
 // 	s.az.CloseFile(internal.CloseFileOptions{Handle: h})
 // }
 
+func (suite *blockBlobTestSuite) TestTruncateSmallFileToSmaller() {
+	suite.UtilityFunctionTestTruncateFileToSmaller(2*MB, 1*MB)
```

Comment:
I see you made these file sizes significantly smaller than the ones used by BF2.
I assume the intent here is to reduce our cloud storage costs for testing?

Reply:
Yes, that was the intention here.